### PR TITLE
adding North Korea sanctions crawler

### PR DESCRIPTION
**Pull request by bgmello**
Close opensanctions/crawler-planning#143

### CODE REVIEW

**Comment by pudo** [comment]
Nice! Can we maybe rename it to not be called `sanctions`? The data source is sort of insistent they are not. Perhaps: `us_nk_jointventures`? I think we can also a) get rid of the `make_sanction`, turn the `topic` into `trade.risk` and add (`country`, `nk`) to all of them :)

**Comment by bgmello** [comment]
I used the title from the website:

Screenshot 2024-12-03 at 16 22 53

Do you prefer something more general like `North Korea Joint Ventures`, or is it better to specify at least that it is some sort of restriction like `North Korea Trade Restrictions`?

**Comment by pudo** [comment]
Maybe we can reflect it in the user-facing title, e.g. `US North Korea Advisory Joint Ventures Annex` (god that's complicated)? 

The thing I'm trying to reflect is that this list is form an annex with the following heading: 

Annex 2
Joint Ventures in North Korea
NOTE: This annex lists joint ventures that have operated or are currently operating in
North Korea established prior to 2016 and is not a comprehensive list of North Korean
joint ventures that exist at this time. This annex is not a “sanctions list.” While some
entities on this list may be entities whose property and interests in property are
blocked, the inclusion of these entities in this advisory does not constitute as a
determination by OFAC that any of these entities meet the designation criteria under any
sanctions program. Persons subject to sanctions can be found on OFAC’s List of Specially
Designated Nationals and Blocked Person and other sanctions lists, which can be searched
here.

File: datasets/us/nk_jointventures/us_nk_jointventures.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  > NOTE: This annex lists joint ventures that have operated or are currently operating in
+  > North Korea established prior to 2016 and is not a comprehensive list of North Korean
+  > joint ventures that exist at this time. This annex is not a “sanctions list.” While some
+  > entities on this list may be entities whose property and interests in property are
+  > blocked, the inclusion of these entities in this advisory does not constitute as a
+  > determination by OFAC that any of these entities meet the designation criteria under any
+  > sanctions program. Persons subject to sanctions can be found on OFAC's List of Specially
+  > Designated Nationals and Blocked Person and other sanctions lists, which can be searched
+  > here.
```
Comment (jbothma) [inline datasets/us/nk_jointventures/us_nk_jointventures.yaml:28]:
```suggestion
  > here.
```
```suggestion
  > [here](http://sanctionssearch.ofac.treas.gov/).
```

File: datasets/us/nk_jointventures/us_nk_jointventures.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  This dataset contains a list of joint ventures related to North Korea.
+description: |
+  > CBP and ICE, with the U.S. Department of State, and the U.S. Department of the
+  > Treasury's Office of Foreign Assets Control (OFAC), issued the North Korea
+  > Sanctions and Enforcement Actions Advisory: Risks for Businesses with Supply
+  > Chain Links to North Korea on July 23, 2018. The advisory informs the trade
+  > of North Korea's evasion tactics, compliance risks and requirements, as well
+  > as due diligence best practices.
+
+  Note that this is not a sanctions list as stated in the source PDF:
```
Comment (jbothma) [inline datasets/us/nk_jointventures/us_nk_jointventures.yaml:18]:
```suggestion
  This advisory includes a list of joint ventures with partners from China and other countries.
  We make this list of joint ventures available as a dataset of entities with noteworthy trade risk.
  
  The UN Security Council resolution 2375 requires Member States to prohibit all joint
  ventures with North Korean entities and individuals, whether new or already existing, 
  subject to very narrow exceptions.
  
  According to the advisory:
```

File: datasets/us/nk_jointventures/us_nk_jointventures.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+title: US North Korea Advisory Joint Ventures Annex
+entry_point: crawler.py
+prefix: us-nkjv
+coverage:
+  frequency: daily
+  start: "2024-12-03"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  This dataset contains a list of joint ventures related to North Korea.
```
Comment (jbothma) [inline datasets/us/nk_jointventures/us_nk_jointventures.yaml:9]:
```suggestion
  Joint ventures between entities in North Korea and other countries to be considered trade risk.
```

File: datasets/us/nk_jointventures/us_nk_jointventures.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
+title: US North Korea Advisory Joint Ventures Annex
```
Comment (jbothma) [inline datasets/us/nk_jointventures/us_nk_jointventures.yaml:1]:
```suggestion
title: US Advisory on North Korean Joint Ventures
```

**Comment by jbothma** [comment]
Thanks for this!

**Pull request merged**